.github/workflows/ci-regenerate-fixtures.yml

```diff
@@ -0,0 +1,39 @@
+name: Test With Regenerate Fixtures
+
+on:
+  schedule:
+    - cron: '0 15 * * 1' # Every Monday 7AM PST 
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+        with:
+          ref: staging
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.12.1'
+          cache: 'yarn'
+
+      - name: Cache Rust
+        uses: Swatinem/rust-cache@v2
+        with:
+          shared-key: nodejs
+
+      - name: Install packages
+        run: yarn --non-interactive --frozen-lockfile
+
+      - name: Delete fixture files
+        run: node ./tools/delete-fixtures.js
+
+      - name: Run tests
+        run: JEST_TIMEOUT=1000000000 yarn test:coverage --maxWorkers=2 --workerIdleMemoryLimit=2000MB
```

Comment:
What is the `workerIdleMemoryLimit` based on?

Reply:
This is carried over from our other CI test workflows, but I think it's roughly correlated to the amount of memory on GH Actions VMs, with a bit of headroom. https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners

Unfortunately Jest + NodeJS tests seem to leak memory over time and crash, so they added this flag to restart workers when they approach the memory limit.